src/da65.vcxproj

```diff
@@ -86,6 +86,7 @@
     <ClCompile Include="da65\infofile.c" />
     <ClCompile Include="da65\labels.c" />
     <ClCompile Include="da65\main.c" />
+    <ClCompile Include="da65\opc4510.c" />
```

Comment:
That "da65.vcxproj" diff is a Git artifact.  It shouldn't be shown because it was merged by PR #359.